src/components/Button.tsx

```diff
@@ -0,0 +1,35 @@
+import { FontType } from 'styled-components'
+import { BaseButton } from './styles/Button.styled'
+
+type Props = {
+  text: string
+  isActive?: boolean
+  link?: string | undefined
+  htmlType?: 'button' | 'submit' | 'reset' | undefined
+  fontType: FontType
+}
+
+const defaultProps = {
+  isActive: false,
+  link: null,
+  htmlType: 'submit',
+}
+
+function Button(props: Props) {
+  const { isActive } = props
```

Comment:
destructuring not necessary, unless used for all. can just use props.isActive

Reply:
Resolved in commit 14a46b1a4b9ebc8f475728d33fefb789fa93f86d.